jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -997,6 +997,17 @@ public static void load(String pathName) {
  */
 @CallerSensitive
 public static void loadLibrary(String libName) {
+	if (libName.indexOf(File.pathSeparator) >= 0) {
```

Comment:
This set of tests is inconsistent with the one in `load(String)`. `File.isAbsolute()` says these are not:
```
/bar
\bar
X:bar
```
I suggest we create a helper method to be used both places.

Reply:
I don't understand, they are two different things. loadLibrary() doesn't allow paths (absolute or relative), and load() must be absolute on jdk15+.

Reply:
If we have a helper method, say `isAbsolute(String path)`, `load(String)` would insist that it returns `true`, while `loadLibrary(String)` would insist that it return `false`.

Reply:
I think it finally sunk in: The helper I described wouldn't work.